pyaedt/application/AnalsyisEmit.py

```diff
@@ -8,34 +8,31 @@
 
 
 class FieldAnalysisEmit(Design):
-    """**AEDT Emit Analysis**
+    """FieldAnaysisEmit  class.
 
-    Class for Emit Analysis Setup (Emit)
-
-    It is automatically initialized by Application call (like HFSS,
-    Q3D...). Refer to Application function for inputs definition
+    The class is for setting up an EMIT analysis in AEDT.
+    It is automatically initialized by an application call (like HFSS,
+    Q3D, and ...). Refer to the Application function for input definitions.
```

Comment:
```suggestion
    It is automatically initialized by an application call (e.g. HFSS,
    Q3D, etc.). Refer to the Application function for input definitions.
```

Reply:
Rewriting as all Ansys guidelines say to avoid using Latin terms.